mage/installation/cugraph.md

```diff
@@ -0,0 +1,118 @@
+---
+id: cugraph
+title: Install MAGE × NVIDIA cuGraph
+sidebar_label: MAGE × NVIDIA cuGraph
+---
+
+Follow this guide to install Memgraph with [**NVIDIA cuGraph**](https://github.com/rapidsai/cugraph) GPU-powered graph
+algorithms.
+
+### Prerequisites
+
+:::info
+To be able to run cuGraph analytics, make sure you have compatible infrastructure first. The exact system requirements
+are available at the [**NVIDIA RAPIDS site**](https://rapids.ai/start.html#requirements), and include an NVIDIA Pascal 
+(or better) GPU and up-to-date CUDA & NVIDIA drivers.
+:::
+
+**Docker requirements :whale:**
+
+If running MAGE × NVIDIA cuGraph in Docker, the following applies:
+
+- To run on NVIDIA-powered GPUs, RAPIDS requires Docker CE v19.03+ and
+  [**nvidia-container-toolkit**](https://github.com/NVIDIA/nvidia-docker#quickstart) installed.
+- Legacy Docker CE v17-18 users require the installation of the
+  [**nvidia-docker2**](https://github.com/NVIDIA/nvidia-docker/wiki/Installation-(version-2.0)) package.
+
+**Local build requirements:**
+
+If building MAGE × NVIDIA cuGraph locally, these requirements apply (tested on Ubuntu):
+
+- [**CMake**](https://cmake.org/) version above 3.20
+- [**NVIDIA CUDA developer toolkit**](https://developer.nvidia.com/cuda-toolkit) – CUDA version 11.6
+- System dependencies: `libblas-dev`, `liblapack-dev`, `libboost-all-dev`
+- [**NVIDIA NCCL communications library**](https://developer.nvidia.com/nccl)
+
+## Pulling the Docker image from DockerHub
+
+```shell
+docker run -p 7444:7444 memgraph/memgraph-mage:1.3-cugraph-22.02-cuda-11.5
+```
+
+* Depending on your environment, different versions of MAGE/cuGraph/CUDA can be installed:
+```shell
+docker run -p 7444:7444 memgraph/memgraph-mage:${MAGE_VERSION}-cugraph-${CUGRAPH_VERSION}-cuda-${CUDA_VERSION}
+```
+
```

Comment:
Add here maybe:

" To check which versions are available, jump to Docker Hub. There we have few options from which you can choose"